README.md

```diff
@@ -44,6 +44,6 @@ See https://developer.mozilla.org/en-US/docs/Learn/JavaScript/Building_blocks/Ev
 
 If `useCapture` is true, the event will be registered in the capturing phase and thus, propagated top-down instead of bottom-up as is the default.
 
-### display: `PropTypes.oneOf(['block', 'inline-block'])`
+### display: `PropTypes.oneOf(['block', 'flex, 'inline-block'])`
```

Comment:
Not a major deal since it's the README, but you're missing a closing quote on `flex`.